package-deb.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
Shouldn’t need this file if we change this to `native` packaging and actually build from the source code in the git repository.